components/dashboard/src/settings/Preferences.tsx

```diff
@@ -30,6 +30,14 @@ export default function Preferences() {
         const settings = additionalData.ideSettings || {};
         settings.defaultIde = value;
         additionalData.ideSettings = settings;
+        getGitpodService().server.trackEvent({
+            event: "ide_configuration_changed",
+            properties: {
+                useDesktopIde,
+                defaultIde: value,
+                defaultDesktopIde
```

Comment:
I am seeing the `defaultDesktopIde` value set as the last selected value when the `useDesktopIde` is set to false.

**Steps to replicate:** 
* Choose "Open in Desktop IDE"
* Click an IDE, e.g. PHPStorm
* Unselect "Open in Desktop IDE"
* Select VS Code or VS Code insiders

**Expected result:** 
```json
{"properties":{"useDesktopIde":false,"defaultIde":"code"}}
```

**Actual result:**
```json
{"properties":{"useDesktopIde":false,"defaultIde":"code","defaultDesktopIde":"phpstorm"}}
```

The `phpstorm` value is still set / stored in scope, so seems to be set. If the `useDesktopIde` property is false, can we remove the `defaultDesktopIde` property completely?

Reply:
@iQQBot let's go with @loujaybee suggestion to move this PR.